From 90fb7bb85002cde39de3b3d2e2481933390832af Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Fri, 8 Dec 2023 22:35:45 +0000
Subject: [PATCH] configure: set docdir in GTK_DOC_CHECK

By passing --docdir in the GTK_DOC_CHECK arguments (to match
autogen.sh) autoreconf will work out of the box.

Without this autoreconf fails due to the documentation not being in
./docs, the default location.

Upstream-Status: Submitted [https://lore.kernel.org/linux-modules/20231208224511.1363066-1-ross.burton@arm.com/T/#u]
Signed-off-by: Ross Burton <ross.burton@arm.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 7056aae..d53a20c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -236,7 +236,7 @@ AS_IF([test "x$enable_coverage" = "xyes"], [
 AM_CONDITIONAL([ENABLE_COVERAGE], [test "x$enable_coverage" = "xyes"])
 
 m4_ifdef([GTK_DOC_CHECK], [
-GTK_DOC_CHECK([1.14],[--flavour no-tmpl-flat])
+GTK_DOC_CHECK([1.14],[--flavour no-tmpl-flat --docdir libkmod/docs])
 ], [
 AM_CONDITIONAL([ENABLE_GTK_DOC], false)])
 
